content/docs/authorization/deployment/_index.md

```diff
@@ -62,27 +66,30 @@ A Storage Administrator can execute the installer or rpm package as a root user
       "hostName": "DNS_host_name"
```

Comment:
When you say `DNS_host_name` do you mean a fully qualified domain name (FQDN) ?
Also, and sorry to be picky but... `_` are not permitted by the RFC ; only letters, digits, and hyphen. Do you think we can replace the occurrences of `DNS_host_name` by `dns-hostname` or `fqdn` ?

Reply:
fixed